website/docs/articles/appservice-plan-dns-resolution.md

```diff
@@ -0,0 +1,35 @@
+---
+sidebar_label: Ensuring an Online Service is Reachable from an App Service Plan
```

Comment:
By 'online service,' do you also mean external services, or is the article specifically focused on Azure cloud services?

Reply:
Any online service